content/en/apps/guides/performance/telemetry.md

```diff
@@ -72,6 +72,7 @@ Find below the list of telemetry data recorded by CHT:
 | `report_detail:<form>:load` | On the Reports tab, the time taken to load a report from the point it was selected on the left hand side to the time it was fully rendered. Added in 4.7. | Yes. Added in 4.7 |
 | `messages_detail:load` | On the Messages tab, the time taken to load the messages detail on the right hand side once has been selected from the list on the left hand side. Added in 4.7. | Yes. Added in 4.7 |
 | `sidebar_filter:reports:open` | Number of times the user opens the sidebar filter in Reports tab. | |
+| `sidebar_filter:analytics_target_aggregates:open` | Number of times the user opens the sidebar filter in Aggregate Targets tab. | |
```

Comment:
Small consideration... what do you think about renaming this to be consistent with `analytics:<detail_page>:load`.  We'll probably put the filter on the targets page at some point as well (specifically to handle reporting period), so we could have...

`sidebar_filter:analytics:targets:open`
and
`sidebar_filter:analytics:target_aggregates:open`

I recognize this is more than just a docs change. If it's quick and easy, I think we should change it. Otherwise the other one (when we add to personal targets filter) will be `sidebar_filter:analytics_targets:open` which isn't too bad.

Reply:
That's a small change, I will incorporate it as I make the final changes to the PR. 

Once that change is made, I will come back to update this PR

Reply:
I have updated the telemetry in the CHT [Code](https://github.com/medic/cht-core/pull/9267/files#:~:text=this.telemetryService.record(%27sidebar_filter%3Aanalytics%3Atarget_aggregates%3Aopen%27)%3B)  and also made the change in this PR